bugbounty.md

```diff
@@ -1,11 +0,0 @@
-# Bug Bounties with Immunefi
```

Comment:
I think we should leave this file

Reply:
Will do, thank you!

Reply:
Restored in 99f378b0ce4d79f95c0d773e9db5513cc524522d